web/public/templates/dojos/events/user-events.dust

```diff
@@ -24,7 +24,7 @@
             <tr ng-click="showEventInfo(dojoEvents, $index)" class="event-info-row">
               <td>{{ event.name }}</td>
               <td>{{ event.formattedDate }}</td>
-              <td>{{ event.type }}</td>
+              <td>{{ getEventTypeDisplayText(event) }}</td>
```

Comment:
Didn't see it was a ng-repeat, hence the use of a function can be justified while there is no existing, preparing, loop like you have describe. Even better, there is only 2 possibles outputs, so that an key-val object could be used to store the value to avoid consecutive calls to ng-translate 
Finally my remark on binding is that this tpl is display oriented, hence it doesn't make sense to double bind since the model shouldn't change